extensions/data-workspace/package.json

```diff
@@ -69,6 +75,11 @@
           "when": "view == dataworkspace.views.main",
           "group": "secondary"
         },
+        {
+          "command": "dataworkspace.close",
+          "when": "view == dataworkspace.views.main && workbenchState == workspace",
+          "group": "tertiary"
```

Comment:
Since groups order alphabetically, the general guidance from VSCode dev docs is to use numbers (except for the special "navigation" group) to guarantee ordering: https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups

We get a bit lucky since that shakes out nicely with "primary", "secondary", and "tertiary" alphabetizing naturally, but something like `1_currentWorkspace` (for refresh) and `2_commands` (for close) might set a more expandable naming scheme going forward.

Reply:
lucky = I had to sing my ABCs to make sure Tertiary would come after Secondary.
I reworked the menu items, 1_navigation (new and open), 2_currentWorkspace (refresh), and 3_commands (close)